worker/src/cf.rs

```diff
@@ -140,6 +140,11 @@ impl Cf {
         let tz = self.inner.timezone();
         tz.parse::<chrono_tz::Tz>().unwrap()
     }
+
+    /// Whether the country of the incoming request is in the EU
+    pub fn is_eu_country(&self) -> bool {
+        self.inner.is_eu_country() == Some("1")
```

Comment:
Because the sys crate's definition returns an `Option<String>` we can't compare against a string reference in the form of a `Option<&str>`. Comparing to an own `String` would work here though.

```suggestion
        self.inner.is_eu_country() == Some("1".to_string())
```

Reply:
Thanks for the review. Sorry about that! I refactored and forgot to change that back. Should be updated now.